make.js

```diff
@@ -454,6 +452,21 @@ function buildTask(taskName, taskListLength, nodeVersion) {
             rm('-Rf', taskTestsNodeModulesPath);
         }
     }
+
+    // remove duplicated task libs node modules from build tasks.
+    var buildTasksNodeModules = path.join(buildTasksPath, taskName, 'node_modules');
```

Comment:
Removing duplicated task-lib via make.json file would require us to add it under Tasks\taskname\_buildConfigs files for each task and identify all tasks that are having duplicated task-libs. I prefer to remove duplicates via buildTask function.

Reply:
In that case, if you want to remove them during the build, it may be better to handle not only defined common packages, but all node_modules, something similar to [the function](https://github.com/microsoft/azure-pipelines-tasks/blob/master/ci/after-build-check-tasks.js#L81) from ci/cd checks?